### PR TITLE
KJ Outro Event.

### PR DESCRIPTION
**Pull request by Ulduar**
Small additional.
Add missing referrence npc.

### CODE REVIEW

File: src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/sunwell_plateau/boss_kiljaeden.cpp [hunk 1]
```diff
@@ -119,18 +119,20 @@ enum
...
     SPELL_ENTROPIUS_BODY        = 46819,
     SPELL_BLAZE_TO_LIGHT        = 46821,
     SPELL_SUNWELL_IGNITION      = 46822,
+    SPELL_PORTAL_FROM_SHATTRATH = 46801,            // Riftwalker after reached point 1, summon NPC_RIFTWALKER and cast on him this spell.
 
     NPC_INERT_PORTAL            = 26254,
     NPC_CORE_ENTROPIUS          = 26262,
     NPC_SOLDIER                 = 26259,            // summoned in 2 waves before Velen. Should move into 2 circle formations
     NPC_RIFTWALKER              = 26289,
+    NPC_SHATTRATH_PORTAL        = 26251,
```
Comment (xfurry) [inline src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/sunwell_plateau/boss_kiljaeden.cpp]: Are you sure this patch is complete?

I don't see any reference to this NPC anywhere

Comment (Ulduar) [reply src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/sunwell_plateau/boss_kiljaeden.cpp]: Riftwalker on summon, should walk (false) on position, summon npc id 26251 and cast on this npc spell id 46801.
Shattrat_Portal movement on height, meet npc have same entry and despawn. After this actions, summon one npc 26254. Npc 26254 on spawn cast on self 42047 (implemented in SD2) and enjoy!

**Comment by cyberium** [comment]
@xfurry? Close or merge this please. Mabe @killerwife also.

**Comment by killerwife** [comment]
I think that I should be able to merge this one without issues. Its TBC so I will deal with it.

**Pull request closed without merging**